### ISSUE TITLE
The time consuming of rapter

### ISSUE THREAD

**Issue by Liu-lxq** [amonszpart/RAPter#6]
I ran almost three hours of rapter.py under /RAPter/scripts with the scene "Nola" by using the command "python rapter.py -s 0.05". Is this normal?

**Comment by amonszpart** [comment]
Yes, it's solving a few, quite large binary quadratic optimization problems, the 'max' equations are expensive to compute (it's also a 3 years old implementation, could be sped up on the GPU today). Try compiling Bonmin with Pardiso and SuiteSparse's GPU-based matrix algebra toolbox, it can get you 20-30% speedup.
